pkg/expr/sql_command_test.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 func TestNewCommand(t *testing.T) {
+	t.Skip()
```

Comment:
We're skipping this test, right? I'm surprised to see this getting merged into the codebase. Shouldn't we remove this test, or this `Skip` line?